protocol/communities/community.go

```diff
@@ -824,9 +824,8 @@ func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*protobuf.CommunityD
 		}
 	}
 
-	o.removeMemberFromOrg(pk)
-
 	if isControlNode {
+		o.removeMemberFromOrg(pk)
```

Comment:
This is moved into the condition because admins should not be able to remove users like that, We'll need a pending state for this.

For now, we at least prevent the member to be removed by the admin

Reply:
Undoing this as I'm not handling kicking/banning members in this PR.